components/dashboard/Dashboard.tsx

```
@@ -79,87 +78,137 @@ export const Dashboard: React.FC<DashboardProps> = ({
-      if (cleanup) {
-        cleanup()
+    lastFetchRef.current = now
+
+    try {
+      switch (table) {
+        case 'lists':
+          const { data: listsData } = await supabase
+            .from('lists')
+            .select('*')
+            .eq('group_id', group.id)
+            .order('created_at', { ascending: false })
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)